### PR TITLE
Adding PPC platform support

### PR DESCRIPTION
**Pull request by ranade1**
## What changes were proposed in this pull request?

- Enable IBM Power PC platforms such as PPC RHEL8 and PPC RHEL7 support

## How was this patch tested?

- Manual test

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

**Review by wing2fly — APPROVED** [review]

File: Makefile.vars [hunk 1]
```diff
@@ -143,6 +143,7 @@ PYBABEL := $(ROOT)/build/env/bin/pybabel
 # Path to files for pip requirements
 ##############################
 REQUIREMENT_FILE := $(ROOT)/desktop/core/requirements.txt
+REQUIREMENT_PPC64LE_FILE := $(ROOT)/desktop/core/requirements_ppc64le.txt
```
Comment (JohanAhlen) [inline Makefile.vars:146]: Shouldn't the new requirements file be part of this change?

Comment (ranade1) [reply Makefile.vars:146]: It has been submitted to internal GitHub.

**Comment by ranade1** [comment]
recheck

**Comment by ranade1** [comment]
recheck

File: Makefile.vars (hunk 1 above)
Comment (SonalMahambrey1) [reply Makefile.vars:146]: @ranade1 @JohanAhlen  
We were testing build on  ppc64le architecture and build is failing due to missing file desktop/core/requirements_ppc64le.txt .
Also build passed with desktop/core/requirements.txt. So is following change required in makefile which is merged in this PR
@if [[ "ppc64le" == $(PPC64LE) ]]; then \
	  echo '--- Installing $(REQUIREMENT_PPC64LE_FILE) into virtual-env via $(ENV_PIP)'; \
	  $(ENV_PIP) install -r $(REQUIREMENT_PPC64LE_FILE); \
	  echo '--- Finished $(REQUIREMENT_PPC64LE_FILE) into virtual-env'; \

If above change is required we will need desktop/core/requirements_ppc64le.txt.

Kindly suggest

**Pull request merged**